web-console/Storage/OB-OB-Lifecycle-Management/OB-OBC-Lifecycle.md

```diff
@@ -126,8 +97,7 @@ Overview tab:
 * Creation time
 * Status
 * Storage class
-* Capacity data: Usage/ OB quota/ pie chart for available
-* Object Bucket claim that created this claim
+* Object Bucket claim that created this bucket
 
 ![OB details](img/OB-overview.png)
```

Comment:
Same here... should we remove capacity from the design?

Reply:
Can we add capacity to the futures placeholder section so it's not forgotten?

Reply:
The capacity was removed. lines 129/130 were deleted and the design was updated... unless I'm missing something here. We kept "usage" since it's only showing the used capacity without any total or quota. As far as I know, this is possible to implement.

@julienlim Sure, ill add it.

Reply:
@shirimordechay Same here...I see the text updates but not the mockup update. Again, I think we can merge and then follow up if needed on the images :)